content/faq/data/how-is-data-processed.de.md

```diff
@@ -0,0 +1,18 @@
+---
+title: >
+    Wie werden die Daten verarbeitet?
+slug: wie-werden-die-daten-verarbeitet
+weight: 20
+---
+
+Der Überholabstandsmesser lädt ganze Fahrten als einzelne Dateien ins Portal
+hoch. Diese werden importiert, jedoch nur zur Einzelanzeige der jeweiligen
+Fahrt.
+
+Später werden wir durch verschiedene Verarbeitungsalgorithmen abgeleitete
+Datensätze aus mehreren Fahrten erstellen. Diese werden wir so weit es geht
```

Comment:
Was bedeuted Anonymisierung "so weit es geht"? Welche Daten können nicht anonymisiert werden?

Mein Vorschlag wäre, entweder hinschreiben welche Daten nicht anonymisiert werden können oder diesen Zusatz weglassen.

Reply:
> Welche Daten können nicht anonymisiert werden?

Das ist fast schon philosophisch. Im Endeffekt kann ich mit viel Aufwand alle Daten deanonymisieren. Krassestes Beispiel: Ich kann  alle anderen Besitzer:innen von OBS in deiner Region finden, und ihnen viel Geld geben dafür dass sie mir ihre ganzen Dateien geben. Dann kann ich die Differenz zwischen dem öffentlichen Datensatz und meinem bilden, und hab nur noch deine Daten.

Weniger krass ist der Fall, dass du als einziger in einer Region aufzeichnest. Irgendwie komm ich immer an Infos über dich ran, ganz anonym bist du nicht. Aber wir können *so weit es geht* versuchen, das schwer zu machen (durch Aggregation, Maskierung, Reduzieren von Metadaten, ...).

Das geht in eine ähnliche Richtung wie "100% Sicherheit gibt es nicht".

Reply:
Ich schlage vor dass wir in der entsprechenden Dokumentationsseite zum Anonymisierungsverfahren dann genau dokumentieren was passiert, und dann von der FAQ da hin linken für mehr Informationen.

Reply:
@SubOptimal bist Du auch so einverstanden?

Reply:
Die Antwort von @opatut ist mir wohl durchgerutscht.